azext_edge/edge/_help.py

```diff
@@ -1016,19 +1016,81 @@ def load_iotops_help():
         short-summary: Create asset endpoint profiles.
     """
 
+    helps[
+        "iot ops asset endpoint create custom"
+    ] = """
+        type: command
+        short-summary: Create an asset endpoint profile for a custom connector.
+        examples:
+        - name: Create an asset endpoint with anonymous user authentication using the given instance in the same resource group.
+          text: >
+            az iot ops asset endpoint create custom --name myprofile -g myresourcegroup --instance myinstance
+            --target-address http://rest-server-service.azure-iot-operations.svc.cluster.local:80 --endpoint-type rest-thermostat
+        - name: Create an asset endpoint with username-password user authentication using the given instance in a different resource group but same subscription. The additional
+                configuration is provided as an inline json.
+          text: >
+            az iot ops asset endpoint create custom --name myprofile -g myresourcegroup --instance myinstance
+            --instance-resource-group myinstanceresourcegroup
+            --target-address http://rest-server-service.azure-iot-operations.svc.cluster.local:80 --endpoint-type rest-thermostat
+            --username-ref rest-server-auth-creds/username --password-ref rest-server-auth-creds/password
+            --additional-config addition_configuration.json
+        - name: Create an asset endpoint with certificate authentication using the given instance in the same resource group.
+          text: >
+            az iot ops asset endpoint create custom --name myprofile -g myresourcegroup --instance myinstance
+            --target-address http://rest-server-service.azure-iot-operations.svc.cluster.local:80 --endpoint-type rest-thermostat
+            --certificate-ref mycertificate.pem
+        - name: Create an asset endpoint with anonymous user authentication using the given instance in the same resource group. The inline content is a powershell syntax example.
+          text: >
+            az iot ops asset endpoint create custom --name myprofile -g myresourcegroup --instance myinstance
+            --target-address http://rest-server-service.azure-iot-operations.svc.cluster.local:80 --endpoint-type rest-thermostat
+            --additional-config '{\\\"displayName\\\": \\\"myconnector\\\", \\\"maxItems\\\": 100}'
```

Comment:
Rather than enumerating every variation of inline json format per supported shell, and repeating across multiple commands, I suggest we show one inline example for bash (description indicating so), then when the command is used and there are json parsing errors we can append to the error an aka.ms link to our repo wiki page showing all the variations of inline json input across shells.

Reply:
The latest commit updates the help , but this part was not addressed

> then when the command is used and there are json parsing errors we can append to the error an aka.ms link to our repo wiki page

Which I still think we should do when we know inline json was attempted.